code/game/gamemodes/cult/runes.dm

```diff
@@ -340,144 +340,101 @@ var/list/teleport_runes = list()
 		fail_invoke()
 
 
-//Rite of Enlightenment: Converts a normal crewmember to the cult. Faster for every cultist nearby.
+//Rite of Offering: Converts a normal crewmember to the cult or sacrifices mindshielded and sacrifice targets.
 /obj/effect/rune/convert
-	cultist_name = "Rite of Enlightenment"
-	cultist_desc = "converts a normal crewmember on top of it to the cult. Does not work on mindshielded crew."
+	cultist_name = "Rite of Offering"
+	cultist_desc = "Offers a non-cultists on top of it to your deity, either converting or sacrificing them."
 	invocation = "Mah'weyh pleggh at e'ntrath!"
 	icon_state = "3"
-	req_cultists = 2
-
+	req_cultists = 1
+	allow_excess_invokers = 1
+	rune_in_use = 0
+	
 /obj/effect/rune/convert/do_invoke_glow()
 	return
 
 /obj/effect/rune/convert/invoke(var/list/invokers)
-	var/list/convertees = list()
-	var/turf/T = get_turf(src)
-
-	for(var/mob/living/M in T.contents)
-		if(!iscultist(M) && !ismindshielded(M) && !isgolem(M) && ishuman(M))
-			convertees.Add(M)
-	if(!convertees.len)
-		fail_invoke()
-		log_game("Convert rune failed - no eligible convertees")
+	if(rune_in_use)
 		return
-	var/mob/living/new_cultist = pick(convertees)
-	if(!is_convertable_to_cult(new_cultist.mind) || new_cultist.null_rod_check())
-		for(var/M in invokers)
-			to_chat(M, "<span class='warning'>Something is shielding [new_cultist]'s mind!</span>")
-			if(is_sacrifice_target(new_cultist.mind))
-				to_chat(M, "<span class='cultlarge'>\"I desire this one for myself. <i>SACRIFICE THEM!</i>\"</span>")
+	var/list/myriad_targets = list()
+	var/turf/T = get_turf(src)
+	for(var/mob/living/M in T)
+		if(!iscultist(M))
+			myriad_targets |= M
```

Comment:
should not need to be |=. Unless the same target is listen twice or more on the turf for some reason

Reply:
Ah, no idea what |= did, it was just part of how TG Code listed the targets. Thanks.

Reply:
as reference for others. It adds an item to a list if it's not yet in the list. This is the case for lists. For other variables this is a bitwise operation.
+= will do here since the turf SHOULD not have duplicates of mobs.